src/nvim/tui/term_input.inl

```diff
@@ -269,7 +269,11 @@ static TermInput *term_input_new(void)
     flags |= TERMKEY_FLAG_RAW;
   }
 
-  rv->tk = termkey_new_abstract(os_getenv("TERM"), flags);
+  const char *term = os_getenv("TERM");
+  if (!term) {
+    term = "";  // termkey_new_abstract assumes non-null (#2745)
+  }
+  rv->tk = termkey_new_abstract(term, flags);
```

Comment:
OK, it'll only return `NULL` on OOM. First, it tries to construct a [CSI-based driver](https://github.com/neovim/libtermkey/blob/619c0c12828a86dcd388b0915b2e9656d6aac92a/driver-csi.c) with terminfo, but that'll fail so it falls back on a [TI driver](https://github.com/neovim/libtermkey/blob/619c0c12828a86dcd388b0915b2e9656d6aac92a/driver-ti.c). The docs don't say what TI stands for (terminal independent?), but the point being this seems safe.